RELEASE_NOTES.md

```diff
@@ -1,5 +1,81 @@
 # Release Notes
 
+## 5.4.0
+### Features
+- A preview of Podman's support for OCI artifacts has been added through the `podman artifact` suite of commands, including `add`, `inspect`, `ls`, `pull`, `push`, and `rm`. This support is very early and not fully complete, and the command line interface for these tools has not been finalized. We welcome feedback on the new artifact experience through our issue tracker!
```

Comment:
@baude did `add` land Brent?  Or should we not mention it here at the moment and note it later?